### DISCUSSION TITLE
`@nuxtjs/i18n` is not on the official roadmap

### DISCUSSION THREAD

**Discussion by mrleblanc101** [nuxt/nuxt#16368] [Questions]
Any reason why `@nuxtjs/i18n` is not on the official roadmap ?
But there is `nuxt/translations` on the roadmap which is only the documentation translation and not the translation module itself ?
`@nuxtjs/i18n` is an essential module for many people.
I know it's a community module, but it's still under the `@nuxtjs` namescpace and is as important as `@nuxtjs/auth` imho which is on the roadmap.

**Comment by kurashina-design** [comment]
https://v8.i18n.nuxtjs.org/

**Reply by mrleblanc101** [reply]
Yes, v8 is not out yet that's why you'd want to show it on the official roadmap like the other packages.

**Comment by pi0** [comment]
The roadmap in docs is focused on features we plan to partially add to either core or built-in dependencies.

@kazupon is hardly working on i18n support Check [intlify/vue-i18n-next](https://github.com/intlify/vue-i18n-next) and [intlify/nuxt3](https://github.com/intlify/nuxt3).

**Reply by mrleblanc101** [reply]
@nuxtjs/auth is not core ?
I18n and Auth are as important ? Why is Auth on the official roadmap

**Reply by kazupon** [reply]
Hello! 👋

The current status of @nuxtjs/18n is v8 beta.
I'm communicating with the Nuxt team to work on the implementation.

In my opinion, I think it will not include i18n in the core. because i18n is really hard and there are many things to consider.

As you can see from the new Nuxt documentation, Nuxt has a lot of features, such as routing, SEO & meta, assets, data fetching and etc. 
For all of these, we have to take into account almost a multiplicative number of patterns, even if you only take i18n.

So, Fortunately, Nuxt has module system called Nuxt module. 
Nuxt module is awesome! it's a loosely coupled module system that is complete in its own module. 

I will continue to work with the Vue community and the Nuxt community through vue-i18n and my project [Intlify](https://github.com/intlify).
I want more to do for the i18n OSS project via Vue and Nuxt ecosystems.

I believe that Nuxt team will continue to support me in the future :)

**Reply by mrleblanc101** [reply]
I understand all that, but the roadmap include many modules like nuxtjs/auth, nuxtjs/image and nuxtjs/content which are not part of the core 🤔 

Anyway I just think it would offer a better visibility. It might not be part of the core but it's essential for many so having one place to check instead of looking at all the separate repo for their own roadmap would still be great.

**Reply by mrleblanc101** [reply]
I see the distinction now, it's `nuxt/image` and not `nuxtjs/image`. Si it's an official module.

I see `nuxtjs/auth` will become `nuxt/auth` with the next release.

Sorry for the confusion.
Although i18n is not an official official module, it's still essential for many and would be great to have a better visibility.

I remember having some issue with `nuxtjs/auth` redirect because it did not properly support i18n routing (probably because i18n is not part of the core, so they did not plan for i18n redirect)
